test/config/monitoring/monitoring.yaml

```diff
@@ -43,7 +43,7 @@ spec:
     spec:
       containers:
       - name: zipkin
-        image: docker.io/openzipkin/zipkin:2.13.0
+        image: docker.io/openzipkin/zipkin:2.23.2
```

Comment:
Newer version is always good as long as it does not break anything. Do you notice any improvement?

Reply:
Yep, agreed :)
The UI is improved in the new version, however thats of secondary importance. 
I haven't noticed any specific API/performance changes. This seemed like an simple step to help eliminate/isolate causes for #4744